src/System.Security.Permissions/src/System/Security/Permissions/FileDialogPermission.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace System.Security.Permissions
+{
+    public sealed partial class FileDialogPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        public FileDialogPermission(System.Security.Permissions.FileDialogPermissionAccess access) { }
+        public FileDialogPermission(System.Security.Permissions.PermissionState state) { }
+        public System.Security.Permissions.FileDialogPermissionAccess Access { get { return default(System.Security.Permissions.FileDialogPermissionAccess); } set { } }
+        public override System.Security.IPermission Copy() { return default(System.Security.IPermission); }
```

Comment:
We should consider having all of the Copy methods just "return this;" rather than "return default(...)".  That way at least it'll hand back a non-null object.